src/main/java/com/couchbase/intellij/tree/PillowFightDialog.java

```diff
@@ -0,0 +1,1507 @@
+
+        } else {
+            try {
+                if (Integer.parseInt(lockTextField.getText()) < 0) {
+                    errors.add("Lock value must be 0 or greater");
+                }
+            } catch (NumberFormatException e) {
+
+            }
+        }
+
+        if (subdocComboBox.getItem() == "enable" && jsonComboBox.getItem() == "disable") {
+            errors.add("Subdoc must be used with JSON");
+        }
+
+        if (pathcountTextField.getText() == null || pathcountTextField.getText().trim().isEmpty()) {
+
+        } else {
+            try {
+                if (Integer.parseInt(pathcountTextField.getText()) < 0) {
+                    errors.add("Pathcount must be 0 or greater");
+                } else if (!pathcountTextField.getText().equals("") && jsonComboBox.getItem() == "disable") {
+                    errors.add("Pathcount must be used with JSON");
+                }
+            } catch (NumberFormatException e) {
+
+            }
+        }
+
+        if (errors.isEmpty()) {
+            super.doOKAction();
+            try {
+                PillowFightCommand(String.valueOf(bucketComboBox.getSelectedItem()), String.valueOf(durabilityComboBox.getSelectedItem()), persistToTextField.getText(), batchSizeTextField.getText(), numberItemsTextField.getText(), keyPrefixTextField.getText(), numberThreadsTextField.getText(), percentageTextField.getText(), String.valueOf(noPopulationComboBox.getSelectedItem()), String.valueOf(populateOnlyComboBox.getSelectedItem()), minSizeTextField.getText(), maxSizeTextField.getText(), numberCyclesTextField.getText(), String.valueOf(sequentialComboBox.getSelectedItem()), startAtTextField.getText(), String.valueOf(timingsComboBox.getSelectedItem()), expiryTextField.getText(), replicateToTextField.getText(), lockTextField.getText(), String.valueOf(jsonComboBox.getSelectedItem()), String.valueOf(noopComboBox.getSelectedItem()), String.valueOf(subdocComboBox.getSelectedItem()), pathcountTextField.getText());
+            } catch (IOException | InterruptedException e) {
+                throw new RuntimeException(e);
+            }
+        } else {
+            errorMessage.setText("<html>" + errors.stream().collect(Collectors.joining("<br>")) + "</html>");
+        }
+    }
+
+    public void PillowFightCommand(String selectedBucket, String selectedDurability, String selectedPersistToTextField, String batchSizeTextField, String numberItemsTextField, String keyPrefixTextField, String numberThreadsTextField, String percentageTextField, String noPopulation, String populateOnly, String minSizeTextField, String maxSizeTextField, String numberCyclesTextField, String sequential, String startAtTextField, String timings, String expiryTextField, String replicateToTextField, String lockTextField, String json, String noop, String subdoc, String pathcountTextField) throws IOException, InterruptedException {
```

Comment:
methods in java don't usually start with an uppercase letter.

Reply:
Also, please move the code that runs PillowFight to its own class. As a rule of thumb, if your class has more than 600 lines, something is wrong.